### PR TITLE
Update README.md Fixes #6

### LINKED ISSUE #6: I wanna add Contributing.md file in the project for open source contribution
Hey @Vin205 

Currently your project lacks contributing.md files please assign this issue under gssoc to me

### PR DESCRIPTION
**Pull request by rishyym0927**
Fixes #6

Hey I have updated the readme.md

### CODE REVIEW

**Comment by rishyym0927** [comment]
fixed

File: README.md [hunk 1]
```diff
@@ -1,79 +1,145 @@
...
+   ```bash
+   git checkout -b feature/your-feature-name
+   ```
+3. Make your changes and commit them:
+   ```bash
+   git commit -m "Add your detailed message here"
+   ```
+4. Push to your branch:
+   ```bash
+   git push origin feature/your-feature-name
```
Comment (skmirajulislam) [inline README.md:99]: it should branch name of remote repo

File: README.md [hunk 2]
```diff
@@ -1,79 +1,145 @@
...
-- [Create React App Documentation](https://facebook.github.io/create-react-app/docs/getting-started)
-- [React Documentation](https://reactjs.org/)
+## 📥 How to Contribute
+
+We welcome contributions to the Enyanjyoti platform! To get started:
+
+1. Fork the repository.
+2. Create a new branch with a descriptive name:
+   ```bash
+   git checkout -b feature/your-feature-name
```
Comment (skmirajulislam) [inline README.md:91]: this one is your current branch, update it

**Review by skmirajulislam — COMMENTED** [review]
change in mentioned part

**Review by skmirajulislam — CHANGES_REQUESTED** [review]
update the pr as mentioned!

**Pull request merged**